src/main/java/com/google/vinet/servlets/LoginServlet.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
```

Comment:
Copyright notice should probably be updated to 2020

Reply:
thanks for spotting this, fixed now https://github.com/googleinterns/step226-2020/pull/6/commits/13d9990c83ddd8bb14716788ffcb6a0c84e9dad2